src/autorenderloop.js

```diff
@@ -143,9 +143,16 @@ olcs.AutoRenderLoop.prototype.enable = function() {
     that.notifyRepaintRequired();
   };
 
-  // Listen for changes on the layer group
-  this.ol3d.getOlMap().getLayerGroup().on('change',
+  // Listen for changes on the main layer group
+  this.ol3d.getOlMap().getLayerGroup().on('change:layers',
       this._boundNotifyRepaintRequired);
+
+  this.ol3d.getOlMap().getLayers().forEach((layer) => {
+    // Listen for changes on layer group
+    layer.on('change', that._boundNotifyRepaintRequired);
+    // Listen for property changes on simple layer (opacity, visibility, ...)
+    layer.on('propertychange', that._boundNotifyRepaintRequired);
+  });
```

Comment:
Yesterday I tried out removing the `if` in the OpenLayers code and it was responsive in OL3-Cesium. Looking at the OpenLayers code it looks more consistent to do the change in OpenLayers: https://github.com/openlayers/openlayers/pull/7073.